tutorials/xr/openxr_settings.rst

```diff
@@ -233,6 +248,36 @@ If your game only supports controllers this should be turned off.
 
 See the chapter on :ref:`hand tracking <doc_openxr_hand_tracking>` for additional details.
 
+Hand Tracking Unobstructed Data Source
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Enabling this means hand tracking uses the exact position of fingers, usually
+what a headset camera sees.
+
+Hand Tracking Controller Data Source
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Enabling this means hand tracking will use the controller itself, and infer where
```

Comment:
This is about what data source the application wants the OpenXR runtime to use, but it might not be supported on the current runtime or hardware, so I feel like "may" is better than "will":

```suggestion
Enabling this means hand tracking may use the controller itself, and infer where
```